code/game/turfs/simulated/floor.dm

```diff
@@ -100,7 +100,9 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 	icon_state = pick(broken_states)
 	broken = 1
 
-/turf/simulated/floor/proc/burn_tile()
+/turf/simulated/proc/burn_tile()

```

Comment:
Can you move this proc from `turfs/simulated/floor.dm` to `turfs/simulated.dm` for consistency?